hypershift-operator/controllers/hostedcluster/internal/platform/platform.go

```diff
@@ -49,6 +51,11 @@ type Platform interface {
 	ReconcileSecretEncryption(ctx context.Context, c client.Client, createOrUpdate upsert.CreateOrUpdateFN,
 		hcluster *hyperv1.HostedCluster,
 		controlPlaneNamespace string) error
+
+	// GetPlatformSpecificPolicyRules responsible to return list of policy rules are required to be used
+	// by the CAPI provider in order to manage the resources by this platform
+	// Return nil if no aditional policy rule is required
+	GetPlatformSpecificPolicyRules() []rbacv1.PolicyRule
```

Comment:
Please, let's rename this to `CAPIProviderPolicyRules` so it's specific and consistent with the other naming e.g `CAPIProviderDeploymentSpec`

Reply:
Done